### PR TITLE
fix: add url box and width distribution

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/6330
Screenshot from 2021-01-12 15-59-41

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Decrease public URL width more

File: app/templates/components/ui-table/cell/cell-link.hbs [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+  </UiPopup>
+  <a 
+    id="public_url"
+    href="{{this.record}}" 
+    target="_blank" rel="noopener">
+    <div  class="ui blue icon button ml-auto rounded-l-none m-0">
+      <i class="share icon" ></i>
+    </div>
+  </a>
+</div>
```
Comment (iamareebjamal) [inline app/templates/components/ui-table/cell/cell-link.hbs:16]: This component is being used a lot, make it a new component and reuse here and on video table

**Comment by maze-runnar** [comment]
Screenshot from 2021-01-12 16-41-43

File: app/components/url-box.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import Component from '@glimmer/component';
+
+export default class UrlBox extends Component {}
```
Comment (iamareebjamal) [inline app/components/url-box.js]: No need of empty component, as discussed previously

File: app/templates/components/ui-table/cell/cell-link.hbs [hunk 3]
```diff
@@ -1,16 +1 @@
...
-  <a 
-    id="public_url"
-    href="{{this.record}}" 
-    target="_blank" rel="noopener">
-    <div  class="ui blue icon button ml-auto rounded-l-none m-0">
-      <i class="share icon" ></i>
-    </div>
-  </a>
-</div>
+<UrlBox @sourceUrl={{this.record}}/>
```
Comment (iamareebjamal) [inline app/templates/components/ui-table/cell/cell-link.hbs:1]:
```suggestion
<UrlBox @url={{this.record}} />

```

File: app/templates/components/url-box.hbs [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<div class="ui action input" style="width: 100%; height: 36px">
+  <input class="truncate" style="flex: 1 0 !important" type="text" value={{@sourceUrl}} placeholder={{@sourceUrl}} readonly>
```
Comment (iamareebjamal) [inline app/templates/components/url-box.hbs:2]: Change to url from sourceUrl

File: app/templates/components/ui-table/cell/cell-link.hbs [hunk 5]
```diff
@@ -1,3 +1 @@
-<a href="{{this.record}}">
-  {{this.record}}
-</a>
+<UrlBox @url={{this.record}}/>
```
Comment (iamareebjamal) [inline app/templates/components/ui-table/cell/cell-link.hbs:1]:
```suggestion
<UrlBox @url={{this.record}} />

```

**Pull request merged**